build.sbt

```diff
@@ -13,13 +14,17 @@ lazy val root = (project in file("."))
   .settings(DefaultBuildSettings.defaultSettings(): _*)
   .settings(SbtDistributablesPlugin.publishingSettings: _*)
   .settings(inConfig(Test)(testSettings): _*)
+  .settings(HeaderPlugin.autoImport.headerSettings(IntegrationTest))
   .configs(IntegrationTest)
-  .settings(inConfig(IntegrationTest)(Defaults.itSettings): _*)
+  .settings(inConfig(IntegrationTest)(Defaults.itSettings ++
+    AutomateHeaderPlugin.autoImport.automateHeaderSettings(IntegrationTest)): _*)
   .settings(
-    Keys.fork in IntegrationTest := false,
+    javaOptions += "-Dlogger.resource=logback-test.xml",
     unmanagedSourceDirectories in IntegrationTest := (baseDirectory in IntegrationTest)(base => Seq(base / "it")).value,
+    resourceDirectory in IntegrationTest :=(baseDirectory in IntegrationTest)(base => base / "it" / "resources").value,
     parallelExecution in IntegrationTest := false,
-    javaOptions += "-Dlogger.resource=logback-test.xml"
+    Keys.fork in IntegrationTest := true,
+    addTestReportOption(IntegrationTest, "int-test-reports")
```

Comment:
sbt-auto-build already provides an integrationTestSettings https://github.com/hmrc/sbt-settings/blob/master/src/main/scala/uk/gov/hmrc/DefaultBuildSettings.scala#L70 - only different line you have is Keys.fork in IntegrationTest set to true rather than false (can still utilise it to make build.sbt cleaner)